### ISSUE TITLE
Error when install with cython

### ISSUE THREAD

**Issue by liug0324** [glample/fastBPE#28]
running install
running bdist_egg
running egg_info
writing fastBPE.egg-info\PKG-INFO
writing dependency_links to fastBPE.egg-info\dependency_links.txt
writing top-level names to fastBPE.egg-info\top_level.txt
package init file 'fastBPE\__init__.py' not found (or not a regular file)
reading manifest file 'fastBPE.egg-info\SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'fastBPE.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg
running install_lib
running build_py
running build_ext
building 'fastBPE' extension
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.22.27905\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MT -IfastBPE -IC:\Users\liug\Anaconda2\envs\py3\include -IC:\Users\liug\Anaconda2\envs\py3\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.22.27905\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.22.27905\include" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.7.2\include\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /EHsc /TpfastBPE/fastBPE.cpp /Fobuild\temp.win-amd64-3.7\Release\fastBPE/fastBPE.obj -std=c++11 -Ofast -pthread
cl: 命令行 warning D9025 :正在重写“/Os”(用“/Ot”)
cl: 命令行 warning D9002 :忽略未知选项“-std=c++11”
cl: 命令行 warning D9002 :忽略未知选项“-Of”
cl: 命令行 warning D9002 :忽略未知选项“-Oa”
cl: 命令行 warning D9002 :忽略未知选项“-pthread”
fastBPE.cpp
F:\lab\写作辅助项目\fastBPE-master\fastBPE\fastBPE.hpp(15): fatal error C1083: 无法打开包括文件: “sys/mman.h”: No such file or directory
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.22.27905\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
Dose the file mman.h exist in the github package?

**Comment by akx** [comment]
No, and it shouldn't. It looks like fastBPE isn't Windows compatible at this point.
